### PR TITLE
User show page #11

### LINKED ISSUE #11: User Story 16, User Show Page
```
As a Visitor,
When I click on a user's name for any book review
I am taken to a show page for that user.
I should see all reviews that this
user has written.
Each review shows:
- the title of the review
- the description of the review
- the rating of the review
- the title of the book
- the thumbnail image for the book
- the date the review was written
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #12

child of #33

depends on #16

### PR DESCRIPTION
**Pull request by Vjp888**
Creates a user show page to list out reviews.

closes #11

### CODE REVIEW

File: app/views/reviews/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<h1><%= @user %></h1>
+
+<% @reviews.each do |review|%>
+<section id="review-<%= review.id %>">
+  <img src="<%= review.book.thumbnail %>" alt="cover on <%= review.book.title %>">
+  <p>Title of Book: <%= review.book.title %></p>
```
Comment (milevy1) [inline app/views/reviews/index.html.erb]: Can you make the thumbnail and book title links to that book show page?

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
+        <%= @book.grab_reviews('top', 3).each do |review| %>
+          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username, user_index_path(review.username) %></p>
         <% end %>
       </section>
 
       <section class="bottom-reviews">
-        <p>Bottom Reviews</p>
-        <% @book.grab_reviews('bottom', 3).each do |review| %>
-          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username %></p>
+        <p>Bottome Reviews</p>
```
Comment (milevy1) [inline app/views/books/show.html.erb]: Can you update lines 23-24? I corrected these typos in a previous PR but looks like they got missed here.
L23 - Bottom
L24 - Just the equal sign on the erb tag

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -14,15 +14,15 @@
     <% unless @book.reviews.empty? %>
       <section class="top-reviews">
         <p>Top Reviews</p>
-        <% @book.grab_reviews('top', 3).each do |review| %>
-          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username %></p>
+        <%= @book.grab_reviews('top', 3).each do |review| %>
```
Comment (milevy1) [inline app/views/books/show.html.erb]: Remove this erb equal sign too, I got it on a previous PR

File: app/views/books/show.html.erb (hunk 2 above)
Comment (Vjp888) [reply app/views/books/show.html.erb]: Took care of the typo and removed the errant echo tag.

File: app/views/books/show.html.erb (hunk 3 above)
Comment (Vjp888) [reply app/views/books/show.html.erb]: Removed per request, I might have missed these when I handled a merge conflict.

**Comment by milevy1** [comment]
Looks good, I approve to merge.
